nextflow_schema.json

```diff
@@ -33,6 +33,13 @@
                     "description": "Sequencing center information to be added to read group of BAM files.",
                     "fa_icon": "fas fa-synagogue"
                 },
+                "read_length": {
+                    "type": "integer",
+                    "description": "Read length to calculate macs genome size (--macs_gsize)parameter",
+                    "fa_icon": "fas fa-chart-area",
+                    "default": 100,
```

Comment:
This isn't actually the default, the default is `null` currently. So it failed out of the box for me.